compiler/testData/ir/irText/expressions/implicitCastOnPlatformType.fir.txt

```diff
@@ -2,5 +2,6 @@ FILE fqName:<root> fileName:/implicitCastOnPlatformType.kt
   FUN name:test visibility:public modality:FINAL <> () returnType:kotlin.String
     BLOCK_BODY
       RETURN type=kotlin.Nothing from='public final fun test (): kotlin.String declared in <root>'
-        CALL 'public open fun getProperty (p0: kotlin.String?): kotlin.String? declared in java.lang.System' type=kotlin.String? origin=null
-          p0: CONST String type=kotlin.String value="test"
+        TYPE_OP type=kotlin.String origin=IMPLICIT_NOTNULL typeOperand=kotlin.String
+          CALL 'public open fun getProperty (p0: kotlin.String?): kotlin.String? declared in java.lang.System' type=kotlin.String? origin=null
```

Comment:
parameter `p0`, return type `kotlin.String?` need `@FlexibleNullability`, but no idea where to look at yet...

Reply:
Citing myself:
```
Also, why this check `if (isJavaPlatformClass)` is needed? As I know, all Java types with flexible nullability should have this annotation...
```